.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I'm on a Mac, this hidden file gets created automatically. Adding it to .gitignore keeps repo clean for everyone else.